addons/l10n_fi/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# coding=utf-8
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+# Copyright (C) RockIT Oy 2014-
+# Copyright (C) Avoin.Systems 2014-
+# Authors:
+#   Miku Laitinen (Avoin.Systems)
+#   Mikko Närjänen (Avoin.Systems)
+#   Toni Pesola (Avoin.Systems)
+#   Mikko Salmela (RockIT Oy)
+
+# noinspection PyStatementEffect
+{
+    "name": "Finland - Accounting",
+    "category": "Localization",
+    "version": "1.0.0",
+    "author": "RockIt Oy & Avoin.Systems",
+    "depends": [
+        "account",
+        "base_vat",
+        "base_iban",
+        "l10n_multilang",
```

Comment:
Because the taxes are translated. "ALV" on an invoice won't mean a lot to foreign customers but "VAT" will, and vice versa.